database/migrations/2023_12_20_063541_create_addresses_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('addresses', function (Blueprint $table) {
+            $table->unsignedBigInteger('user_id')->primary();
```

Comment:
```suggestion
            $table->foreignId('user_id')->constrained();
```
外部キーは大抵この書き方で指定します。
あと->primary()を使うとこのカラムがプライマリーキー（テーブルのレコードを一意に識別する値）となるのであまり良くありません。

Reply:
ご指摘ありがとうございます。
外部キーを設定する際に心掛けます。